WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -101,12 +108,24 @@ class PostsListActivity : AppCompatActivity(),
     }
 
     private fun setupContent() {
```

Comment:
I don't want to double comment on a subject, but this is very important, we should not have logic in the activity if we can help it. Any usage of `SiteModel` is a no-no for me especially since we don't keep track of the updates to the `SiteModel` in the activity.

@malinajirka Could you remove the `SiteModel` global property from this activity if possible?

Reply:
>@malinajirka Could you remove the SiteModel global property from this activity if possible?

Yep, I agree we should do this.